Grace0950 proposal for HyperTS

grace0950 opened the pull request:
<!-- Please fill the following checkboxes -->
---
A good PR should include the following parts:

- [x] Name your branch like ``<github-user-name>-hw#``, e.g., ``myname-hw2-attempt1``
- [x] Provide a clear title
- [x] Provide meaningful descriptions in the PR and the commit logs
- [x] Merge from your specific branch (***not master***) to the `hw*` branch (***not master***, again)
- [x] Exclude irrelevant files, such as binaries, text files, or dot files
- [x] Make sure tests/CI pass

yungyuc reviewed (changes requested):
Leave a global comment when you are ready for the next review.  Please add an entry on E3.  I am not seeing it.

- [ ] Follow [`proposal/README.rst`](https://github.com/yungyuc/nsdhw_25sp/blob/f83e66ee5f60a614fbe5614581d5d09ced2c344d/project/README.rst?plain=1#L73) to add your proposal in it.
- [ ] Obey the 79-char line width or it is hard for me to review.
- [ ] Don't add test at the last phase of your development.  Do it early and revise the plan accordingly.
- [ ] Make a short and memorable project name.

grace0950 commented:
@yungyuc I have made all the requested changes:

- Renamed the project to HyperTS.
- Updated the schedule to integrate testing early in the process.
- Added the proposal to proposal/README.rst, following the 79-character line rule.

yungyuc commented on project/grace0950/README.rst:
```diff
@@ -0,0 +1,241 @@
+      -  Develop FFT computation module in C++.
+      -  Implement Kalman Filter with multi-threading support.
+      -  Develop ARIMA model in C++.
+      -  Unit testing for each module as it is implemented.
```
You should have added an inline annotation to highlight your change here.  Please take care of it in the future.

It wastes time for a reviewer to scan through the document or changesets to find your modification.  A reviewer may need to review dozens of code reviews everyday.  The author should be responsible for making reviews easier.

Read more: https://x.com/yungyuc/status/1883482569582530899

grace0950 replied:
Got it! I'll keep that in mind next time.

yungyuc reviewed (approved):
LGTM.  Let me know when you want it to be merged.

grace0950 commented:
@yungyuc I'm good to merge. Thanks!
